docs_user/modules/proc_decommissioning-rhosp-standalone-ceph-NFS-service.adoc

```diff
@@ -54,19 +43,19 @@ spec:
 __EOF__
 
 ----
-+
+
+. Patch the `OpenStackControlPlane` custom resource:
 ----
-oc patch openstackcontrolplane openstack --type=merge --patch-file=~/manila.patch
+$ oc patch openstackcontrolplane openstack --type=merge --patch-file=~/manila.patch
 ----
 
-. To cleanup the standalone ceph nfs service from the {OpenStackShort} control plane
-nodes, you can disable and delete the pacemaker resources associated with the
-service. Replace `<VIP>` in the following commands with the IP address assigned
-to the ceph-nfs service in your environment.
+. Optional: To clean up the standalone `ceph-nfs` service from the {OpenStackShort} control plane nodes, you can disable and delete the Pacemaker resources associated with the service:
```

Comment:
I added "Optional" to step 3. Can you confirm that this is true?

Reply:
This section ("Decommissioning the Standalone Ceph-NFS service") is about getting rid of the service; so we wouldn't want it to be Optional.

We've already clarified above that this step can be done any time after RHOSO 18 is operational, but must not be delayed forever...